frontend/src/routes/Home/Overview/overviewDataFunctions.tsx

```diff
@@ -0,0 +1,398 @@
+}
+
+export function getApplicationList(
+  apps: Application[],
+  argoApps: ArgoApplication[],
+  allClusters: Cluster[],
+  placementDecisions: PlacementDecision[],
+  subscriptions: Subscription[],
+  filteredClusterNames: string[]
+) {
+  const appList: Application[] = []
+  const localCluster = allClusters.find((cls) => cls.name === 'local-cluster')
+  apps.forEach((application) => {
+    const clusterList = getClusterList(
+      application,
+      argoApps,
+      placementDecisions,
+      subscriptions,
+      localCluster,
+      allClusters
+    )
+    if (clusterList.filter((cluster) => filteredClusterNames.includes(cluster))) {
+      appList.push(application)
+    }
+  })
+  return appList
+}
+
+export function getApplicationCount(
+  filteredClusterNames: string[],
+  appSets: ApplicationSet[],
+  applicationList: Application[],
+  filteredOCPApps: Record<string, any>
+) {
+  const ocpAppsOnSelectedCluster = Object.values(filteredOCPApps).filter((ocpApp) =>
+    filteredClusterNames.includes(ocpApp.cluster)
+  )
+  return [...ocpAppsOnSelectedCluster, ...applicationList, ...appSets].length
+}
+
+export function getPolicyReport(policyReports: PolicyReport[], filteredClusters: Cluster[]) {
+  let policyReportCriticalCount = 0
+  let policyReportImportantCount = 0
+  let policyReportModerateCount = 0
+  let policyReportLowCount = 0
+  let clustersWithIssuesCount = 0
+  const policyReportsForFilteredClusters = policyReports.filter((policyReport: PolicyReport) =>
+    filteredClusters.find((cluster: Cluster) => cluster.name === policyReport.scope?.name)
+  )
+  policyReportsForFilteredClusters.forEach((policyReport: PolicyReport) => {
+    const insightsFilteredResults = policyReport.results.filter((result) => result.source === 'insights')
+    insightsFilteredResults.length > 0 && clustersWithIssuesCount++
+    insightsFilteredResults.forEach((result: PolicyReportResults) => {
+      switch (result.properties.total_risk) {
+        case '4':
+          policyReportCriticalCount++
+          break
+        case '3':
+          policyReportImportantCount++
+          break
+        case '2':
+          policyReportModerateCount++
+          break
+        case '1':
+          policyReportLowCount++
+          break
+      }
+    })
+  })
+
+  return {
+    policyReportCriticalCount,
+    policyReportImportantCount,
+    policyReportModerateCount,
+    policyReportLowCount,
+    clustersWithIssuesCount,
+  }
+}
+
+export function getClustersSummary(
+  filteredClusters: Cluster[],
+  filteredClusterNames: string[],
+  managedClusterInfos: ManagedClusterInfo[],
+  applicationList: Application[],
+  appSets: ApplicationSet[],
+  filteredOCPApps: Record<string, any>,
+  t: TFunction<string, undefined>
+) {
+  const kubernetesTypes = new Set()
+  const regions = new Set() // regions could be retrieved from Node labels?
+  filteredClusters.forEach((curr: Cluster) => {
+    kubernetesTypes.add(curr.labels?.vendor ?? 'Other')
```

Comment:
Is the `Other` value only used for grouping?

Reply:
Yes, this is how the current Overview page groups for Kube types & regions. If the cluster does not have a label for vendor or region it is classified as other.